specification/containerservice/Fleet.Management/fleetnamespace.tsp

```diff
@@ -159,13 +373,32 @@ model FleetManagedNamespaceStatus {
 @doc("A fleet managed namespace.")
 @added(Versions.v2025_03_02_naar)
 model FleetManagedNamespaceProperties {
```

Comment:
Please follow the tradition of other tsp files. We always define data structures top down rather than bottom up.

https://github.com/Azure/azure-rest-api-specs/blob/8a733a2c8bfc1fbb76601e6e6d67566694f4c031/specification/containerservice/Fleet.Management/fleetmember.tsp


Start with something like, and then define the properties.

@doc("A member of the Fleet. It contains a reference to an existing Kubernetes cluster on Azure.")
@resource("members")
@parentResource(Fleet)
model FleetMember is ProxyResource<FleetMemberProperties> {